src/status_im/mailserver/core.cljs

```diff
@@ -290,7 +291,8 @@
                           (log/info "mailserver: messages request success for topic " topics "from" from "to" to)
                           (do
                             (log/error "mailserver: messages request error for topic " topics ": " error)
-                            (re-frame/dispatch [:mailserver.callback/request-error (i18n/label :t/mailserver-request-error-title)])))))))
+                            (js/setTimeout #(re-frame/dispatch [:mailserver.callback/resend-request {:request-id "failed-request"}])
```

Comment:
why not use the set-timeout in utils here?

Reply:
forgot about it :)